itou/eligibility/models.py

```diff
@@ -73,9 +73,7 @@ def has_considered_valid(self, job_seeker, for_siae=None):
         Hence the Trello #2604 decision: if a PASS IAE is valid, we do not
         check the presence of an eligibility diagnosis.
         """
-        return job_seeker.approvals_wrapper.has_valid or bool(
-            self.last_considered_valid(job_seeker, for_siae=for_siae)
-        )
+        return job_seeker.has_valid_approval or bool(self.last_considered_valid(job_seeker, for_siae=for_siae))
```

Comment:
Cette ligne de code corrige par ailleurs[ l'erreur Sentry connue ici](https://sentry.io/organizations/itou/issues/2966334762/?project=6164438&query=is%3Aunresolved) et qui était liée au fait parfois, l'utilisateur n'avait pas de `approvals_wrapper`.